content/docs/guides/neon-authorize-auth0.md

```diff
@@ -141,16 +141,18 @@ Now that you’ve integrated Auth0 with Neon Authorize, you can securely pass JW
 
 ### 1. Add Row-Level Security policies
 
-Below are examples of RLS policies for a **todos** table, designed to restrict access so that users can only create, view, update, or delete their own todos.
+Here's how to implement RLS policies for a **todos** table using a `crudPolicy`. This policy ensure users can only access and modify their own todos.
```

Comment:
Let's keep the language explaining these are examples only. We don't want people thinking they need to copy-paste these samples as part of the mandatory setup. Also, the drizzle tab is using crudPolicy, but the SQL tab is still using granular RLS.

"Here are examples of implementing RLS policies for a **todos** table – the Drizzle example leverages the simplified `crudPolicy` function, while the SQL example demonstrates the use of individual RLS policies."

Reply:
updated